mermithid/processors/TritiumSpectrum/FakeDataGenerator.py

```diff
@@ -298,13 +375,15 @@ def generate_unbinned_data(self, Q_mean, mass, ROIbound, S, B_1kev, nsteps=10**4
         logger.info('Stepsize is {} eV'.format(step_size))
 
         #Options of kinetic energies to be sampled
-        self.Koptions = np.arange(Kmin_eff, Kmax_eff, step_size)
+        self.Koptions = np.arange(Kmin_eff, Kmax_eff, self.lineshape_stepize)
 
         if efficiency_dict is not None:
             logger.info('Evaluating efficiencies')
-            efficiency_mean, efficiency_error = efficiency_from_interpolation(self.Koptions, efficiency_dict, B_field)
+            """efficiency_mean, efficiency_error = efficiency_from_interpolation(self.Koptions, efficiency_dict, B_field)
```

Comment:
delete commented efficiency sampling. the returned efficiencies are already sampled.